fork/.editorconfig

```diff
@@ -0,0 +1,15 @@
+# Analyzers
```

Comment:
Oh, is the  `.editorconfig` still needed if `AnalysisMode` is `None`?

Reply:
yeah... it still fail even with `AnalysisMode` as `None`